pkg/bot/slack_cloud.go

```diff
@@ -211,6 +212,12 @@ func (b *CloudSlack) start(ctx context.Context) error {
 	b.setFailureReason("")
 	go b.startMessageProcessor(ctx, messageWorkers, messages)
 
+	if err := b.reporter.ReportBotEnabled(b.IntegrationName(), b.commGroupMetadata.Index); err != nil {
+		b.setFailureReason(health.FailureReasonConnectionError)
```

Comment:
It's analytics, I would skip setting failure reason here. We can fix that also in socket slack code.

Reply:
I essentially copied the same code we have across all our bots. Should I adjust the code everywhere? 😀